.dockerignore

```diff
@@ -5,7 +5,6 @@ terraform
 .DS_Store
 .next
 .vscode
-.env
```

Comment:
`.env` should be ignored by Docker because for security we don't want secret values stored by accident in an image.